.github/workflows/docker-build-static.yml

```diff
@@ -330,29 +328,60 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-
-      - name: Build and push latest
+          
+######################
+#### latest build ####
+######################
+      - name: Build and push latest alpine
         if: github.event_name != 'release'
         uses: docker/build-push-action@v3
         with:
           context: .
           push: ${{ github.event_name != 'pull_request' }}
-          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          platforms: linux/amd64,linux/arm64
           file: ./.github/workflows/Dockerfile.ci.alpine
           tags: nitnelave/lldap:latest
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache-new
+          
+      - name: Build and push latest debian
+        if: github.event_name != 'release'
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: ${{ github.event_name != 'pull_request' }}
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
+          file: ./.github/workflows/Dockerfile.ci.debian
+          tags: nitnelave/lldap:latest-debian
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new
+          
+#######################
+#### release build ####
+#######################        
+      - name: Build and push release alpine
+        if: github.event_name == 'release'
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          # Tag as latest, stable, semver, major, major.minor and major.minor.patch.
+          file: ./.github/workflows/Dockerfile.ci.alpine
+          tags: nitnelave/lldap:stable, nitnelave/lldap:v${{ steps.slug.outputs.version-semantic }}, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}.${{ steps.slug.outputs.version-minor }}, nitnelave/lldap:v${{ steps.slug.outputs.version-major }}.${{ steps.slug.outputs.version-minor }}.${{ steps.slug.outputs.version-patch }}
```

Comment:
I was thinking, just to be clear, it could be nice to have this be both `:latest` and `:latest-alpine`, WDYT?

Reply:
So the tags become like this?
```
:latest (amd64,arm64)
:latest-alpine (amd64,arm64))
:latest-debian (amd64,arm64,armhf)
```

```
:stable (amd64,arm64)
:stable-alpine (amd64,arm64))
:stable-debian (amd64,arm64,armhf)
```

Reply:
Yes, exactly.

Reply:
And the semver tag become like this?
```
:v0.4.0-debian (amd64, arm64, armhf)
:v0.4.0-alpine (amd64, arm64)
:v0.4.0 (amd64, arm64)
```

Reply:
Yes!